### PR TITLE
documentation update - Powerscale 3.7.0

### PR DESCRIPTION
**Pull request by Saksham-Nautiyal**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [ ] I have performed Ansible Sanity test using --docker default
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Review by sachin-apa — DISMISSED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,14 @@ Dellemc.Powerscale Change Logs
 
 .. contents:: Topics
 
+v3.7.0
+======
+
+Minor Changes
+-------------
+
+- Added check and diff mode support for synciqpolicy module.
```
Comment (trisha-dell) [inline CHANGELOG.rst:13]: Add bug fix for sync IQ policy module.
Applicable to all such instances

Comment (Saksham-Nautiyal) [reply CHANGELOG.rst:13]: done

**Review by sachin-apa — APPROVED** [review]

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

File: docs/RELEASE_NOTES.md [hunk 2]
```diff
@@ -82,8 +82,7 @@ This section describes the features of the Ansible Modules for Dell PowerScale f
 
 The Ansible Modules for Dell PowerScale release 3.4.0 supports the following features:
```
Comment (ABHISHEK-SINHA10) [inline docs/RELEASE_NOTES.md:83]: I can see release version is 3.4.0 here, do we need to update this too ?

**Review by sachin-apa — APPROVED** [review]

File: docs/RELEASE_NOTES.md (hunk 2 above)
Comment (Saksham-Nautiyal) [reply docs/RELEASE_NOTES.md]: done

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

**Review by Kritika-Bhateja-03 — DISMISSED** [review]

File: docs/RELEASE_NOTES.md [hunk 3]
```diff
@@ -80,10 +80,9 @@ New Features and Enhancements
 ---------------------------
 This section describes the features of the Ansible Modules for Dell PowerScale for this release.
 
-The Ansible Modules for Dell PowerScale release 3.4.0 supports the following features:
+The Ansible Modules for Dell PowerScale release 3.7.0 supports the following features:
 
-- Added diff mode support for NFS module.
-- Fixed failure of listing support_assist details in info module for oneFS version 9.4.0 and below.
+- Added check and diff mode support for synciqpolicy module.
```
Comment (trisha-dell) [inline docs/RELEASE_NOTES.md:85]: add note for fixes as well.

**Review by sachin-apa — APPROVED** [review]

File: docs/RELEASE_NOTES.md (hunk 3 above)
Comment (Saksham-Nautiyal) [reply docs/RELEASE_NOTES.md:85]: done

**Review by trisha-dell — APPROVED** [review]

**Review by ABHISHEK-SINHA10 — APPROVED** [review]

**Pull request merged**